Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,23 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+function gryffindorLivers(){
+  console.log("people who belong to the Gryffindor house:")    //display the names of the people who belong to the Gryffindor house
+for({firstName,house,lastName} of hogwarts){
```

Comment:
There is a better way to iterate over the array (native array methods in JavaScript). Please, think about the alternative way to achieve the same goal. Please, don't forget about proper spacing in your code.